src/v/config/node_config.cc

```diff
@@ -155,6 +155,20 @@ node_config::node_config() noexcept
       "Enables log messages for allocations greater than the given size.",
       {.visibility = visibility::tunable},
       std::nullopt)
+  , use_storage_failure_injection(
```

Comment:
Prefer to do noun-verb style names so that both properties start with storage_failure_injection and would get alphabetically sorted together.

Something like "storage_failure_injection_enable"?

I was also wondering about the use case of a separate bool property rather than just relying on null-ness of the path property?

Reply:
> Something like "storage_failure_injection_enable"?

Sure, let's go for that.

> I was also wondering about the use case of a separate bool property rather than just relying on null-ness of the path property?

Decoupling the two makes things more flexible, since the failure config is only parsed at start-up. For instance, it allows for starting with failure injection disabled (warm-up period) and enabling it later on. I plan to add a commit to toggle failure injection via admin api. That will work by setting the boolean node config.

Reply:
Updated the config name to  `storage_failure_injection_enabled` and added a debug route to toggle failure injection at runtime.